### DISCUSSION TITLE
React - How to dynamically disable a single item inside of Carbon Dropdown component

### DISCUSSION THREAD

**Discussion by dhebert-dev** [carbon-design-system/carbon#7780] [Help] [answered]
Hello. I am trying to dynamically disable a single item inside of the Carbon Dropdown component in React. What would be the best approach? I have noticed in the carbon storybook that there is a disable property but it seems to only disable the entire dropdown. I have also tried putting the items inside of an array, giving the item I want disabled the disable=true property and then passing it to the Dropdown component but that did not work. Thanks for your help!

**Comment by tw15egan** [comment, marked as answer]
Hey there @dhebert-dev, it doesn't seem like disabling an individual item is currently supported (Rel https://github.com/carbon-design-system/carbon/issues/6477). As a workaround, you can use the `Select` element, which allows disabling a specific `SelectItem`
